cpu/esp_common/Makefile.include

```diff
@@ -104,14 +51,10 @@ ifneq (,$(filter esp_qemu,$(USEMODULE)))
   CFLAGS += -DQEMU
 endif
 
+# NOTE: These checks can be turned into normal conditionals when #9913 is fixed
 # use 32 priority levels if any WiFi interface or the ETH interface is used
-ifneq (,$(filter esp_wifi esp_now esp_eth,$(USEMODULE)))
-  CFLAGS += -DSCHED_PRIO_LEVELS=32
-endif
-
-ifneq (,$(filter esp_wifi esp_now esp_eth,$(USEMODULE)))
-  INCLUDES += -I$(RIOTCPU)/esp_common/esp-wifi
-endif
+CFLAGS += $(if $(filter esp_wifi esp_now esp_eth,$(USEMODULE)),-DSCHED_PRIO_LEVELS=32)
+INCLUDES += $(if $(filter esp_wifi esp_now esp_eth,$(USEMODULE)),-I$(RIOTCPU)/esp_common/esp-wifi)
```

Comment:
```suggestion
CFLAGS += $(if $(filter esp_wifi_any esp_eth,$(USEMODULE)),-DSCHED_PRIO_LEVELS=32)
INCLUDES += $(if $(filter esp_wifi_any esp_eth,$(USEMODULE)),-I$(RIOTCPU)/esp_common/esp-wifi)
```
should also work here. For documentation it might be better to leave it separated.

Reply:
Changed